torchgeo/datasets/bigearthnet.py

```diff
@@ -504,3 +505,74 @@ def _extract(self, filepath: str) -> None:
         """
         if not filepath.endswith(".csv"):
             extract_archive(filepath)
+
+    def _onehot_labels_to_names(
+        self, label_mask: np.ndarray  # type: ignore[type-arg]
+    ) -> List[str]:
+        """Gets a list of class names given a label mask.
+
+        Args:
+            label_mask: a boolean mask corresponding to a set of labels or predictions
+
+        Returns
+            a list of class names corresponding to the input mask
+        """
+        labels = []
+        for i, mask in enumerate(label_mask):
+            if mask:
+                if self.num_classes == 19:
+                    labels.append(self.classes_19[i])
+                elif self.num_classes == 43:
+                    labels.append(self.classes_43[i])
+        return labels
+
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+
+        Raises:
+            ValueError: if ``self.bands`` is "s1"
```

Comment:
Could we not plot a Sentinel-1-only image? I'm not sure what it would look like, but it doesn't have to be RGB in order to make a plot.

Reply:
Plotting S1 would just be single-channel grayscale. The reason we are paying special attention to RGB is to give users the ability to see part of the data easily (e.g. with some sort of normalization that makes sense). I don't think it would be possible to support visualizing everything for all datasets